client/src/App.js

```diff
@@ -4,6 +4,8 @@ import Home from "./components/Home/Home";
 import NavBar from "./components/NavBar/NavBar";
 import { Routes, Route } from 'react-router-dom';
 import ProductDetails from "./components/ProductDetails/ProductDetails";
+import Login from "./components/SignIn_SignUp/Login/Login";
+import SignUp from "./components/SignIn_SignUp/SignUp/SignUp";
```

Comment:
```suggestion
import SignUp from "./components/Auth/SignUp/SignUp";
```
Nôm na 2 thứ này sẽ được gọi là Auth - Authentication